backend/prompt_studio/prompt_studio_core/urls.py

```diff
@@ -79,9 +77,7 @@
         ),
         path(
             "prompt-studio/index-document/<uuid:pk>",
-            method_decorator(transaction.non_atomic_requests)(
-                prompt_studio_prompt_index
-            ),
+            prompt_studio_prompt_index,
```

Comment:
Will there be any issues if we set the newly added environment variable to `True` after removing this?

Reply:
The indexing operation will be under  transaction. Hence the side effects of the same will be applicable.